### PR TITLE
[CBRD-25200] Add system parameter 'memory_monitoring'

### PR DESCRIPTION
**Pull request by Rudeus**
http://jira.cubrid.org/browse/CBRD-25200

Add system parameter 'memory_monitoring'.

Using in configuration file
memory_monitoring = (y/n)

+) minor change : add cscope.files in .gitignore

### CODE REVIEW

File: conf/cubrid.conf [hunk 1]
```diff
@@ -76,4 +76,4 @@ log_max_archives=0
 # for deduplicate (-1 ~ 14, -1 forces explicit statements to also be ignored.)
 deduplicate_key_level=-1
 print_index_detail=no
-
+memory_monitoring=n
```
Comment (hornetmj) [inline conf/cubrid.conf]:
```suggestion
memory_monitoring=no
```

Comment (hornetmj) [reply conf/cubrid.conf]: For consistency with print_index_detail=`no`

**Review by hornetmj — APPROVED** [review]

File: conf/cubrid.conf (hunk 1 above)
Comment (wrlawodms) [reply conf/cubrid.conf]: Is it a spec to set it NO explicitly though it's by default? If it's not, I think we should remove this.

File: src/base/system_parameter.c [hunk 2]
```diff
@@ -737,6 +737,8 @@ static const char sysprm_ha_conf_file_name[] = "cubrid_ha.conf";
 
 #define PRM_NAME_ORACLE_STYLE_DIVIDE "oracle_style_divide"
 
+#define PRM_NAME_MEMORY_MONITORING "memory_monitoring"
```
Comment (wrlawodms) [inline src/base/system_parameter.c]: How about this to say it a flag.
```suggestion
#define PRM_NAME_MEMORY_MONITORING "enable_memory_monitoring"
```

**Review by wrlawodms — APPROVED** [review]
Approval with some opinions.

File: src/base/system_parameter.c (hunk 2 above)
Comment (hornetmj) [reply src/base/system_parameter.c]: > Is it a spec to set it NO explicitly though it's by default? If it's not, I think we should remove this.

He is right. I think it would be good to romove it.

File: conf/cubrid.conf (hunk 1 above)
Comment (Rudeus) [reply conf/cubrid.conf]: Done. I remove it.

File: src/base/system_parameter.c (hunk 2 above)
Comment (Rudeus) [reply src/base/system_parameter.c]: Done. I remove it. And I think your suggestion "enable_memory_monitoring" is better than "memory_monitoring". I change it.

File: conf/cubrid.conf [hunk 3]
```diff
@@ -76,4 +76,3 @@ log_max_archives=0
 # for deduplicate (-1 ~ 14, -1 forces explicit statements to also be ignored.)
 deduplicate_key_level=-1
 print_index_detail=no
-
```
Comment (joohok) [inline conf/cubrid.conf:79]: unrelated change

**Review by joohok — APPROVED** [review]
